dashboard/modules/reporter/reporter_agent.py

```diff
@@ -300,10 +316,40 @@ def _get_all_stats(self):
         }
 
     @staticmethod
-    def _record_stats(stats):
+    def _record_stats(stats, cluster_stats):
         records_reported = []
-
         ip = stats["ip"]
+        redis_address = stats["redis_address"]
+
+        # -- Instance count of cluster --
+        if "autoscaler_report" in cluster_stats and str(ip) == str(
+                redis_address):
+            active_nodes = cluster_stats["autoscaler_report"]["active_nodes"]
+            for node_type, active_node_count in active_nodes.items():
+                records_reported.extend([
```

Comment:
this is equivalent to records_reported.append(Record(...))

Reply:
changed